client/client.go

```diff
@@ -454,3 +454,18 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	}
 	return results, nil
 }
+
+type DeviceUser struct {
```

Comment:
Indeed, good catch. I unified it to `User` in the client code and to `userResponseData` in the daemon.